src/DependencyInjection/SonataClassificationExtension.php

```diff
@@ -105,28 +105,30 @@ private function configureAdmin(array $config, ContainerBuilder $container): voi
      */
     private function registerSonataDoctrineMapping(array $config): void
     {
-        foreach ($config['class'] as $type => $class) {
+        $classes = $config['class'];
+        foreach ($config['class'] as $class) {
             if (!class_exists($class)) {
                 return;
```

Comment:
We should throw an exception if the class does not exist

Reply:
Didn't want to change the behavior because I don't know this package but I thought about the same thing